src/RequestObjects.ts

```diff
@@ -139,12 +139,7 @@ export class OnboardWithThingIDRequest {
  * @prop {Object} metadata Key-value list to store within command definition.
  */
 export class PostCommandRequest {
-    public aliasActions: Array<AliasAction>;
     public issuer?: string;
-    public title?: string;
-    public description?: string;
-    public metadata?: Object;
-
```

Comment:
> Why we can remove above member fields?

Typescript provides a convenient way to define attribute of class together with constructor. https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/